### PR TITLE
Make oppai-java build easier with build scripts

### LINKED ISSUE #53: make oppai-java build easy
it's not nice to make people copypaste huge gcc/msvc strings to compile oppai-java. we should have a simple build.sh and build.bat script that does it for the most commonly used compilers like node-oppai and pyoppai do.

@dpisdaniel

### PR DESCRIPTION
**Pull request by dpisdaniel**
Closes #53

### CODE REVIEW

File: java-oppai/src/build.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+cxxflags="$CXXFLAGS"
+
+if [ $(uname) = "Darwin" ]; then
+    brew_prefix=$(brew --prefix)
```
Comment (Francesco149) [inline java-oppai/src/build.sh]: this is misaligned and it triggers me pls fix

File: java-oppai/src/build.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	cxxflags="$cxxflags -I/System/Library/Frameworks/JavaVM.framework/Versions/CurrentJDK/Headers"
+	cxxflags="$cxxflags -I/Developer/SDKs/MacOSX10.6.sdk/System/Library/Frameworks/JavaVM.framework/Versions/A/Headers"
+    ${CXX:-clang++}                        	\
+        -I$brew_prefix/opt/openssl/include 	\
+        -L$brew_prefix/opt/openssl/lib     	\
+		$cxxflags							\
```
Comment (Francesco149) [inline java-oppai/src/build.sh]: also misaligned

File: java-oppai/src/build.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        Build.cpp                          	\
+        -lm -lstdc++                       	\
+        -lcrypto                           	\
+        -o liboppai.jnilib
+else
+	cxxflags="$cxxflags -I/usr/lib/jvm/java-8-openjdk-amd64/include"
+	cxxflags="$cxxflags -I/usr/lib/jvm/java-8-openjdk-amd64/include/linux"
+    ${CXX:-g++}              	\
+        -std=c++98           	\
+		-fPIC                	\
```
Comment (Francesco149) [inline java-oppai/src/build.sh]: also misaligned, check that you arent mixing tabs with spaces

**Review by Francesco149 — COMMENTED** [review]
some alignment issues

File: java-oppai/src/build.sh (hunk 2 above)
Comment (Francesco149) [reply java-oppai/src/build.sh]: also there a space after that \ which would break the script

**Comment by dpisdaniel** [comment]
Seems to be fixed now lul

**Pull request merged**